README.md

```diff
@@ -147,13 +147,17 @@ See full list of events: https://codemirror.net/doc/manual.html#events
   value={this.state.text}
   onChange={this.handleChange}
   events={{
-    'blur': (e) => {},
-    'focus': (e) => {},
+    'blur': {this.onBlur},
+    'focus': {this.onFocus},
     //... Add any codeMirror events
   }}
 />
 ```
 
+If changed, the event handlers are reinstalled on component update. Make sure to use instance
```

Comment:
That's correct, but in the example above object will be different every time the component rerenders, so it will definitely cause all event handlers to reinstall on any rerender of the parent, so events need to be wrapped in `useMemo` along with `useCallback` for event handlers, so only when they are really changed this object will be recreated, otherwise, they'll keep changing.
And, yeah, they'll keep reinstalling themselves on `value` change too, so the comparison algorithm should be more sophisticated than what it's now.